### PR TITLE
debug start delay

### PR DESCRIPTION
**Pull request by noblepaul**
https://app.shortcut.com/fullstory/story/174231/investigate-the-solr-restart-delay

### CODE REVIEW

**Comment by hiteshk25** [comment]
Thanks @noblepaul. CCing @patsonluk as well

File: solr/core/src/java/org/apache/solr/core/CoreContainer.java [hunk 1]
```diff
@@ -750,12 +750,14 @@ public void load() {
 
     zkSys.initZooKeeper(this, cfg.getCloudConfig());
     if (isZooKeeperAware()) {
+      log.info("initializing PKIAuth");
       pkiAuthenticationPlugin = new PKIAuthenticationPlugin(this, zkSys.getZkController().getNodeName(),
           (PublicKeyHandler) containerHandlers.get(PublicKeyHandler.PATH));
       // use deprecated API for back-compat, remove in 9.0
       pkiAuthenticationPlugin.initializeMetrics(
           solrMetricsContext.metricManager, solrMetricsContext.registry, solrMetricsContext.tag, "/authentication/pki");
       TracerConfigurator.loadTracer(loader, cfg.getTracerConfiguratorPluginInfo(), getZkController().getZkStateReader());
```
Comment (patsonluk) [inline solr/core/src/java/org/apache/solr/core/CoreContainer.java:759]: Just to be exhaustive : Can we add a log line here for `initializing Tracer` as well?

Comment (patsonluk) [inline solr/core/src/java/org/apache/solr/core/CoreContainer.java:760]: Just to be exhaustive : Can we add `initializing Tracer` here as well? (so we can tell the timing between PKIAuth and trace) :)

Comment (hiteshk25) [reply solr/core/src/java/org/apache/solr/core/CoreContainer.java:760]: @patsonluk please update the PR appropriate. We want to deploy this on staging today. Thanks!

Comment (patsonluk) [reply solr/core/src/java/org/apache/solr/core/CoreContainer.java:760]: Added in https://github.com/fullstorydev/lucene-solr/pull/132/commits/c21c4c182396f25496c16d97eacba647ba394764

**Review by hiteshk25 — APPROVED** [review]
LGTM

**Review by patsonluk — COMMENTED** [review]
LGTM . added 1 extra logging line

**Pull request merged**